projects/element-ng/status-counter/si-status-counter.component.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+import { NgClass } from '@angular/common';
+import { booleanAttribute, Component, computed, input, numberAttribute } from '@angular/core';
+import { SiIconComponent } from '@siemens/element-ng/icon';
+
+@Component({
+  selector: 'si-status-counter',
```

Comment:
Need to support the alias (old) as well.

```suggestion
  selector: 'si-status-counter, si-icon-status',
```